consensus/state.go

```diff
@@ -521,6 +521,13 @@ func (cs *ConsensusState) updateToState(state *sm.State) {
 		lastPrecommits = cs.Votes.Precommits(cs.CommitRound)
 	}
 
+	// Log whether this node is a validator
+	if privVal := cs.privValidator; privVal != nil && validators.HasAddress(privVal.GetAddress()) {
```

Comment:
I thought about it and I think it is better if we do, because otherwise you would have to track the context of each block in the log.
For example, if you wanted to see whether I was a validator for a specific block, I would have to go through the previous log messages until I find one that shows that I was indeed the validator.

Also, once we are moving towards structured logging in JSON I think it will be helpful to be able to easily display graphical logs that show whether you are currently a validator or not.

Actually, do we log whether we are the proposer (if not we should probably add that)?